From 7ab269a1562a4dac3425011b8df7f233c828cd6a Mon Sep 17 00:00:00 2001
From: Jeff Peeler <jpeeler@redhat.com>
Date: Fri, 7 Mar 2014 11:50:58 -0500
Subject: [PATCH] Patch to allow building against python-oslo-sphinx

Recently a renaming occured:
https://bugs.launchpad.net/heat/+bug/1277168

The package with the renaming changed is not yet available.
---
 doc/source/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 5187fca..9d09a10 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -193,7 +193,7 @@ extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.pngmath',
               'sphinx.ext.viewcode',
               'sphinx.ext.doctest',
-              'oslosphinx',
+              'oslo.sphinx',
               'ext.resources']
 
 todo_include_todos = True
